google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -878,6 +878,14 @@ def execute_partition_update sql, params: nil, types: nil,
         #      and all replicas are exhausted without finding a healthy replica,
         #      Spanner will wait for a replica in the list to become available,
         #      requests may fail due to DEADLINE_EXCEEDED errors.
+        # @param [Integer] order_by Configure the order
+        #   that Spanner returns the data in.
+        #   To see the available options refer to
```

Comment:
Nit: you don't have to be so aggressive with the short lines. Our line length standard is either 80 or 120 columns (your choice, we don't really police it that heavily). Shorter like this is technically okay, but looks a little awkward.

Reply:
Oh I thought it was 80 instead of 120, I was trying to keep the sentences together to the best of my ability. But 120 is more than enough. Thanks Danny!